addons/crm/models/crm_lead.py

```diff
@@ -1241,6 +1259,65 @@ def _handle_salesmen_assignment(self, user_ids=False, team_id=False):
     # CLASSIFICATION TOOLS
     # --------------------------------------------------
 
+    def _get_potential_lead_duplicates_domain(self, lead, include_lost=False):
```

Comment:
Why aren't you using self (+ ensure_one to ensure you have a singleton recordset) ? It would allow removing the lead parameter.